### PR TITLE
Update token-etc.json

### PR DESCRIPTION
**Pull request by Adam-Travers**
Adding details for UniversalCoin (UNV) on ETC

### CODE REVIEW

File: tokens/tokens-etc.json [hunk 1]
```diff
@@ -64,5 +64,30 @@
 			"twitter": "",
 			"youtube": ""
 		}
+	},
+	{
+		"symbol": "UNV",
+		"address": "0x6ADa6F48C815689502C43eC1a59F1b5DD3C04E1F",
```
Comment (gamalielhere) [inline tokens/tokens-etc.json]: Submitted address points to a wallet. Not a contract or token.

**Comment by Adam-Travers** [comment]
Just to confirm you are referring to 0x6ADa6F48C815689502C43eC1a59F1b5DD3C04E1F ?

This is indeed our token contract address. Here's a link to the creation # https://gastracker.io/tx/0x033575bdb8cf6bff9659a383cdd98468230a2d68d6cca84e5a137dacb615d792

That contract address works perfectly when adding a custom token in MEW. Please let me know if there's anything else I can do to help. 

Thanks,
Adam

**Comment by gamalielhere** [comment]
@Adam-Travers My apologies. I didn't see this was for ETC.

**Comment by Adam-Travers** [comment]
No problem. Anything else needed from me for this?

**Pull request merged**